src/react/projects/spark-core-react/src/SprkFooter/components/SprkFooterAwards/SprkFooterAwards.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { uniqueId } from 'lodash';
+import PropTypes from 'prop-types';
+import SprkToggle from '../../../SprkToggle/SprkToggle';
+
+class SprkFooterAwards extends Component {
+  constructor(props) {
+    super(props);
+    if (props.awards.images !== undefined) {
+      this.state = {
+        awardsImagesHasIds: props.awards.images.map(
+          item => ({ id: uniqueId(), ...item }),
+        ),
+      };
+    }
+  }
+
+  render() {
+    const { awards } = this.props;
+    if (awards.images === undefined) return false;
+    const { awardsImagesHasIds } = this.state;
+    return (
+      <div className="sprk-o-Stack__item sprk-o-Stack sprk-o-Stack--large">
+        <h3 className="sprk-o-Stack__item sprk-b-TypeBodyOne">
+          {awards.heading}
+        </h3>
+
+        <div className="sprk-o-Stack__item sprk-o-Stack sprk-o-Stack--medium sprk-o-Stack--split@s sprk-u-mbm">
+          {awardsImagesHasIds.map((image) => {
+            const TagName = image.linkElement || 'a';
+            return (
+              <div key={image.id} className="sprk-o-Stack__item">
+                <TagName href={TagName === 'a' ? image.href || '#nogo' : undefined} data-analytics={image.analyticsString}>
+                  <img className={image.addClasses} src={image.src} alt={image.altText} />
+                </TagName>
+              </div>
+            );
+          })}
+        </div>
+
+        <SprkToggle
+          additionalClasses="sprk-o-Stack__item"
+          toggleIconName="chevron-down-circle-two-color"
+          title={awards.disclaimerTitle}
+          titleAddClasses="sprk-b-TypeBodyFour sprk-u-FontWeight--normal"
+          analyticsString={awards.disclaimerAnalytics}
+        >
+          <p className="sprk-b-TypeBodyFour sprk-u-pts sprk-u-pbs">
+            {awards.disclaimerText}
+          </p>
+        </SprkToggle>
+      </div>
+    );
+  }
+}
+
+SprkFooterAwards.defaultProps = {
+  awards: {},
+};
+
+SprkFooterAwards.propTypes = {
+  awards: PropTypes.shape({
```

Comment:
should awards just be required?

Reply:
@RCopeland, The footer is tricky in that regard, because teams could choose not to include the awards or anything else in the footer. The only restriction we have is for the footer to be used at the bottom of the page only.

Reply:
That makes sense for the SprkFooter component, but does it make sense to allow SprkFooterAwards to render without the awards?

Reply:
I see what you mean now. I agree! I'll mark it required. Thanks!